extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -0,0 +1,47 @@
+require([
+	'jquery',
+	'wikia.tracker',
+], function ($, tracker) {
+	'use strict';
+
+	var track = tracker.buildTrackingFunction({
+		action: tracker.ACTIONS.CLICK,
+		category: 'embeddable-discussions',
+		trackingMethod: 'analytics'
+	});
+
+	$(function() {
+		// Track impression
+		track({
+			action: tracker.ACTIONS.IMPRESSION,
+			label: 'embeddable-discussions-loaded',
+		});
+
+		$('.upvote').click(function(event) {
+			var upvoteUrl = event.currentTarget.getAttribute('data-url'),
+				svg = $( event.currentTarget ).children()[0];
```

Comment:
No spaces inside the parentheses per conventions.

Also, you only refer to `svg` by making it a jQuery object with `$(svg)`, so can this just be `$svg = $($(event.currentTarget).children()[0]);`?